public/request/user/update.php

```diff
@@ -27,6 +27,10 @@
 $foundSourceUser = User::whereName($user)->first();
 $foundTargetUser = User::whereName($targetUsername)->first();
 
+if (!$foundSourceUser || !$foundTargetUser) {
+    return back()->withErrors(__('legacy.error.error'));
+}
```

Comment:
This prevents a possible null exception that could occur in the first couple branches below, and moves the logic from the other branches.